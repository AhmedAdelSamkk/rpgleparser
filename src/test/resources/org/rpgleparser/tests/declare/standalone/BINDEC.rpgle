        DCL-S numRecords BINDEC(9);
        DCL-S bonus BINDEC(9:2);