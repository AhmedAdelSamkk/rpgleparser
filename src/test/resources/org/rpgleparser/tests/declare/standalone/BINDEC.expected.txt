/*===TOKENS===*/
DS_Standalone            <DCL-S>
ID                       <numRecords>
ID                       <BINDEC>
'('                      <(>
NUMBER                   <9>
')'                      <)>
FREE_SEMI                <;>
DS_Standalone            <DCL-S>
ID                       <bonus>
ID                       <BINDEC>
'('                      <(>
NUMBER                   <9>
COLON                    <:>
NUMBER                   <2>
')'                      <)>
FREE_SEMI                <;>
/*===TREE===*/
(r
  (dspec
    DCL-S
    (identifier (free_identifier numRecords))
    (datatype
      (identifier (free_identifier BINDEC))
      (args ( (expression (number 9)) ))
    )
    ;
  )
  (dspec
    DCL-S
    (identifier (free_identifier bonus))
    (datatype
      (identifier (free_identifier BINDEC))
      (args ( (expression (number 9)) : (expression (number 2)) ))
    )
    ;
  )
)
/*======*/